fbchat/client.py

```diff
@@ -1519,6 +1650,12 @@ def getThreadIdAndThreadType(msg_metadata):
                         self.onTitleChange(mid=mid, author_id=author_id, new_title=new_title, thread_id=thread_id,
                                            thread_type=thread_type, ts=ts, metadata=metadata, msg=m)
 
+                    # Thread image change
+                    elif delta.get("class") == "ForcedFetch":
```

Comment:
I think `ForcedFetch` happens at other times too, an what it means is basically that the client should fetch information about the thread again. So what we should be doing, is fetch the newest events in the thread, and then take the actor_id and image_id from that. Quite expensive though, in terms of performance, but I think it's the only way?

Reply:
It's the only event that is returned after changing thread image. 😕 When I looked at requests in browser I found that this information is in graphql event. We could try to fetch that. Second option is to find information by action (message) id but I completely don't know how to fetch that and I think that there isn't any possible way to do that.